src/core/audio/AudioSdl.cpp

```diff
@@ -283,24 +304,98 @@ void AudioSdl::sdlInputAudioCallback(Uint8 *_buf, int _len) {
 
 #endif
 
+QString AudioSdl::setupWidget::s_systemDefaultDevice = QObject::tr("[System Default]");
+
 AudioSdl::setupWidget::setupWidget( QWidget * _parent ) :
 	AudioDeviceSetupWidget( AudioSdl::name(), _parent )
 {
 	QFormLayout * form = new QFormLayout(this);
+	form->setRowWrapPolicy(QFormLayout::WrapLongRows);
```

Comment:
Any reasons for this change?

Reply:
Because otherwise the labels might get hidden if the content of the combo boxes becomes very long (which is the case for my sound devices). See commit https://github.com/LMMS/lmms/pull/7421/commits/94b43f5a2f8d2094912362fe14b53bfe26aad6fa.